src/modules/iotjs_module_process.c

```diff
@@ -78,9 +78,28 @@ static jerry_value_t wait_for_source_callback(
 
 // Export JS module received from the debugger client
 JS_FUNCTION(DebuggerGetSource) {
-  jerry_value_t res;
-  jerry_debugger_wait_for_client_source(wait_for_source_callback, NULL, &res);
-  return res;
+  jerry_debugger_wait_for_source_status_t receive_status;
+  jerry_value_t ret_val = jerry_create_array(1);
```

Comment:
Default size is 1?

Reply:
@zherczeg You are right, there are actually some cases where even 1 element initialization is unnecessary. I've fixed it. Thanks for the tip.